commands/conduit_cli.py

```diff
@@ -38,144 +55,117 @@ def get_cluster_arn(app: str, env: str) -> str:
         if app_key_found and env_key_found and cluster_key_found:
             return cluster_arn
 
+    raise NoClusterConduitError
 
-def get_postgres_secret(app: str, env: str, name: str):
-    secret_name = f"/copilot/{app}/{env}/secrets/{name}"
-
-    return boto3.client("secretsmanager").get_secret_value(SecretId=secret_name)
 
+def get_connection_secret_arn(app: str, env: str, name: str) -> str:
+    connection_secret_id = f"/copilot/{app}/{env}/secrets/{name}"
 
-def update_postgres_command(app: str, env: str, command: str, secret_name: str) -> str:
-    secret = get_postgres_secret(app, env, secret_name)
-    secret_arn = secret["ARN"]
-    secret_json = json.loads(secret["SecretString"])
-    postgres_password = secret_json["password"]
+    secrets_manager = boto3.client("secretsmanager")
+    ssm = boto3.client("ssm")
 
-    return f"{command} --secrets DB_SECRET={secret_arn} --env-vars POSTGRES_PASSWORD={postgres_password}"
+    try:
+        return secrets_manager.describe_secret(SecretId=connection_secret_id)["ARN"]
+    except secrets_manager.exceptions.ResourceNotFoundException:
+        pass
 
+    try:
+        return ssm.get_parameter(Name=connection_secret_id, WithDecryption=False)["Parameter"]["ARN"]
+    except ssm.exceptions.ParameterNotFound:
+        pass
 
-def create_task(app: str, env: str, addon_type: str, secret_name: str) -> None:
-    command = f"copilot task run -n tunnel-{addon_type} --image {CONDUIT_IMAGE_LOCATIONS[addon_type]} --app {app} --env {env}"
-    if addon_type == "postgres":
-        command = update_postgres_command(app, env, command, secret_name)
+    raise SecretNotFoundConduitError(name)
 
-    subprocess.call(command, shell=True)
 
+def create_addon_client_task(app: str, env: str, addon_type: str, addon_name: str):
+    connection_secret_arn = get_connection_secret_arn(app, env, addon_name.upper())
 
-def is_task_running(cluster_arn: str, addon_type: str) -> bool:
-    tasks = boto3.client("ecs").list_tasks(
-        cluster=cluster_arn, desiredStatus="RUNNING", family=f"copilot-tunnel-{addon_type}"
+    subprocess.call(
+        f"copilot task run --app {app} --env {env} --task-group-name conduit-{addon_name} "
```

Comment:
Lower case and hyphenate this for consistency, also in the other functions that use addon_name.